integrated_channels/xapi/utils.py

```diff
@@ -59,26 +53,18 @@ def send_course_completion_statement(lrs_configuration, user, course_overview, c
          course_overview (CourseOverview): Course over view object containing course details.
          course_grade (CourseGrade): course grade object.
     """
-    user_details = LearnerInfoSerializer(
-        user,
-        context={'enterprise_customer': lrs_configuration.enterprise_customer}
-    )
-    course_details = CourseInfoSerializer(
-        course_overview,
-        context={'site': lrs_configuration.enterprise_customer.site}
-    )
     LOGGER.info(
```

Comment:
As with the above comment -- do we capture if/when the send fails?

Reply:
yeah capturing it [here](https://github.com/edx/edx-enterprise/blob/877511923e5b2a309bc8cb7a374cdace1fee7073/integrated_channels/xapi/management/commands/send_course_completions.py#L152)